rest_api/objects/index.adoc

```diff
@@ -1339,7 +1339,7 @@ Required::
 
 | `items`
 | xref:../oauth_apis/useroauthaccesstoken-oauth-openshift-io-v1.adoc#useroauthaccesstoken-oauth-openshift-io-v1[`array (UserOAuthAccessToken)`]
-|
+| 
```

Comment:
There are a lot of updates to this file and I am not so sure why the HCP migration would be involved here.

Reply:
Yeah this is not right.